src/os/bluestore/BlueStore.cc

```diff
@@ -7257,7 +7257,8 @@ int BlueStore::_do_read(
   uint64_t offset,
   size_t length,
   bufferlist& bl,
-  uint32_t op_flags)
+  uint32_t op_flags,
+  uint8_t retry_count)
```

Comment:
IMO no much sense in using that narrow int type here, size_t should be fine.

Reply:
I disagree; an actual real checksum error will trigger this always. Really high retry counts will at least generate lots of IOPS or hang the thread. I guess users could guess that lots of retries are always good, so I've restricted the option to 255 specificly.

Or would you just like to change the variable type here without adjusting the value range allowed by the option?

Reply:
Right, I think he's just talking about the variable type.